src/ConnectedMode/Migration/Wizard/MigrationWizardWindow.xaml

```diff
@@ -3,24 +3,41 @@
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:platformUi="clr-namespace:Microsoft.VisualStudio.PlatformUI;assembly=Microsoft.VisualStudio.Shell.15.0"
+    xmlns:vsShell="clr-namespace:Microsoft.VisualStudio.Shell;assembly=Microsoft.VisualStudio.Shell.15.0"
+    
     Title="SonarLint Connected Mode Migration"
     HorizontalAlignment="Center"
     ResizeMode="NoResize"
     SizeToContent="WidthAndHeight"
     MinHeight="400"
-    MaxHeight="600"
```

Comment:
I removed the maximum height so that the dialogue can expand if a large font size is used.

If the user picks a font size that is too large then the dialogue can be taller than the screen. However, this seems to happen with some of the VS dialogues too.